include/deal.II/multigrid/mg_tools.h

```diff
@@ -198,6 +198,18 @@ namespace MGTools
                       const ComponentMask               &component_mask = ComponentMask());
 
   /**
+  * The same function as above, but return an IndexSet rather than a
+  * std::set<unsigned int> on each level and use a std::set of
+  * boundary_indicators as input
+  */
+  template <int dim, int spacedim>
+  void
+  make_boundary_list (const DoFHandler<dim,spacedim>      &mg_dof,
+                      const std::set<types::boundary_id> &boundary_indicators,
```

Comment:
we've recently had a patch where we uniformized the spelling to `boundary_ids`. Maybe do this here as well with the argument name.